packages/xstate-wallet/src/utils/contract-utils.ts

```diff
@@ -24,22 +25,16 @@ export function getProvider(): Web3Provider | JsonRpcProvider {
   if (window.ethereum) {
     return new Web3Provider(window.ethereum);
   } else {
-    return new providers.JsonRpcProvider(`http://localhost:${process.env.GANACHE_PORT}`);
+    return new JsonRpcProvider(`http://localhost:${process.env.GANACHE_PORT}`);
   }
 }
 
 export function getEthAssetHolderContract() {
   const provider = getProvider();
   return new Contract(
+    // eslint-disable-next-line no-process-env
     process.env.ETH_ASSET_HOLDER_ADDRESS || '0x0',
```

Comment:
Is there a reason why a constant from the config cannot be used here?

Reply:
To be honest I’m surprised this is even passing linting. I thought I disabled using process.env outside of the config file.

Reply:
Oh I didn’t even see the disable comment. Yeah let’s not do that :p

Reply:
I'm not sure how it survived linting prior to this PR 😕